.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```

Comment:
Probably that's wrong, if you need some new rules - you can just extend your configuration, but some common rules much better to keep in shared common file